cli/src/utils/createRustVerifyingKey.ts

```diff
@@ -250,20 +251,21 @@ async function createVerifyingKeyRsFile(
         s += "\t],\n\n";
         fs.writeSync(resFile, s);
         s = "\tvk_ic: &[\n";
-        let x = 0;
 
         for (let ic in mydata.IC) {
           s += "\t\t[\n";
           for (let j = 0; j < mydata.IC[ic].length - 1; j++) {
             s += "\t\t\t" + mydata.IC[ic][j] + ",\n";
           }
-          x++;
           s += "\t\t],\n";
         }
         s += "\t]\n};";
         s += appendingString;
 
         fs.writeSync(resFile, s);
+        fs.closeSync(resFile);
+
+        execSync(`rustfmt ${path}`);
```

Comment:
Is this branch properly rebased?

From what I remember, this is my change, which got merged recently, so it should be present on `main`. The fact that Github is showing it as a diff here means that there is something fishy going on.

Reply:
sure, I rebased it with main without leaving issues around, not sure about what git added/removed